docs/strat-lib/guides/approvals.md

```diff
@@ -1,201 +1,106 @@
 
-```mermaid
-sequenceDiagram
-    Maker ->> Maker contract: 
-    rect grey
-    Maker contract ->> Mangrove : Outbound and Inbound
-    end
-    Mangrove ->> Taker: 
-    Taker ->> Mangrove: 
-    Mangrove ->> Maker contract: 
-    Maker contract ->> Maker: 
-```
+### Maker contracts inheriting from `MangroveOffer`
 
-### Making an offer using a Maker contract without a router
+Maker contracts inheriting from [MangroveOffer](../background/offer-maker/mangrove-offer.md) provide two methods to assist with approvals: 
 
-When making an offer using a contract without a router, you only need to approve that the maker contract can transfer the outbound token from the makers [reserve](../background/offer-maker/mangrove-offer#offer-owners-reserve). If the maker didn't set any reserve, then the reserve will be the makers own address. E.g., if you want to make an DAI-USDC offer where you are selling DAI, then you need to approve that the maker contract can take DAI from the makers reserve.
+* [`activate`](../technical-references/code/strategies/MangroveOffer.md#activate) performs the required approvals so as to allow the contract itself to interact with Mangrove on a set of assets.
+* [`checklist`](../technical-references/code/strategies/MangroveOffer.md#checklist) verifies that this contract's current state is ready to be used by `msg.sender` to post offers on Mangrove.
 
-```mermaid
-sequenceDiagram
-    participant Maker as Makers reserve
-    participant MC as Maker contract
-    rect grey
-    Maker ->> MC : Outbound
-    end
-    MC ->> Mangrove : 
-    Mangrove ->> Taker: 
-    Taker ->> Mangrove: 
-    Mangrove ->> MC: 
-    MC ->> Maker: 
 
+### Advanced maker approvals
 
-```
+As drawn, the [call sequence diagram](#call-sequence-overview) depicts a situation where the funds for the smart offer are available directly on the maker contract holding the offer logic. However, for more advanced smart offers several smart contracts may come into play when liquidity is sourced.
 
-### Admin of a Taker contract without a router
+As an outset, the administrator of the maker contract and any other contracts involved in trade settlement initiated by Mangrove must ensure that the proper approvals for liquidity flow is in place before offers are executed. 
 
-When you are an admin of a contract that can take offers, then you need to make sure that the contract itself has made the correct approvals. In this case we are not using a router, so we only need to approve Mangrove to make transfer from the contracts account. E.g., if the contract can take offers on the DAI-USDC market, then you need to makes sure that the contract has approved Mangrove for DAI and USDC transfers.
+For maker contracts building on top of the building blocks provided by the Strat Lib, we comment on two noteworthy advanced making strategies below.
```

Comment:
Ok, sounds good. I've made an update, which says basically this - just linking to the offer owner reserve glossary page, instead of using code-font `reserve`. At the same time, I've improved the Glossary-page for reserve.